src/platform/win32localfilesystem.cpp

```diff
@@ -126,10 +127,11 @@ void Win32LocalFileSystem::Get_File_List_From_Dir(Utf8String const &subdir,
                         Get_File_List_From_Dir(filepath, dirpath, filter, filelist, search_subdirs);
                     }
                 }
-            } while (FindNextFileW(hndl, &data) != 0);
-        }
+            } while (FindNextFileW(hndl, &data) != FALSE);
 
-        FindClose(hndl);
+            // #BUGFIX Moved into scope to avoid close call on invalid handle.
+            FindClose(hndl);
+        }
```

Comment:
We should create an issue for the other side of the ifdef. It can all be done use std::filesystem cross platform and utf8 safe.